front/src/components/QrReader/QrReaderContainer.tsx

```diff
@@ -126,15 +150,26 @@ function QrReaderContainer({ onSuccess }: IQrReaderContainerProps) {
     [checkLabelIdentifier, navigate, triggerError, onSuccess],
   );
 
+  useEffect(() => {
+    checkCameraPermission();
+  },[])
+
   return (
-    <QrReader
+    <>
+      {isCameraNotPermited && <>
+        <AlertWithoutAction alertText="Camera permission was not allowed. Please unblock your camera in the address bar, reload the page, and then grant permission to your camera." />
```

Comment:
I could move the copy to a `const` at the top of the file if this is a more palatable code style (I usually do that.)

```ts
const CAMERA_NOT_PERMITED_TEXT = "Camera permission was not allowed. Please unblock your camera in the address bar, reload the page, and then grant permission to your camera.";
// ...
<AlertWithoutAction alertText={CAMERA_NOT_PERMITED_TEXT}  />
```